### ISSUE TITLE
vertical_parameter_alignment_on_call shouldn't trigger on trailing closure

### ISSUE THREAD

**Issue by marcelofabri** [realm/SwiftLint#1635]
From https://github.com/realm/SwiftLint/issues/1630#issuecomment-309608237:

```swift
transition(
    from: fromController,
    to: toController,
    duration: 0.75,
    options: options.union([.curveEaseInOut]),
    animations: {
        toController.view.snp.makeConstraints { make in
            make.edges.equalToSuperview()
        }
    }
) { _ in
    fromController.removeFromParentViewController()
    toController.didMove(toParentViewController: self)
    self.changeRightBarButtonItem(to: newRightButton)
}
```
